Assets/Scripts/Controllers/InputController.cs

```diff
@@ -143,7 +151,7 @@ public void ExitInteractableObject(InteractableObject interactableObject)
         interactableList.Remove(interactableObject);
     }
 
-    private void Update()
+    protected override void LocalUpdate()
```

Comment:
Current system means we can't interact with objects when paused. May need to add functionality to allow this in the future if needed.